schema_based_on_diagram.sql

```diff
@@ -0,0 +1,55 @@
+CREATE TABLE patients (
+	id SERIAL PRIMARY KEY,
+	name VARCHAR(120),
+	date_of_birth DATE
+);
+CREATE INDEX name_idx ON patients(name);
```

Comment:
- [optional] According to the project requirement we are to create index on foreign keys, having many unnecessary index(es) may reduce the performance of the query. ℹ️ **You may remove it.**